tests/test_optim.py

```diff
@@ -67,12 +67,67 @@ def test_optim(backend, source, spec, mu):
 
     optim = pyhf.optimizer
 
-    result = optim.unconstrained_bestfit(
-        pyhf.infer.utils.loglambdav, data, pdf, init_pars, par_bounds
+    result = optim.minimize(pyhf.infer.mle.loglambdav, data, pdf, init_pars, par_bounds)
```

Comment:
I'm confused as to how this is passing given that

```
$ git checkout errorapi
$ git pull
$ git grep "loglambdav" src/
```

returns no results. Shouldn't these be using the `fit` API now? Also why do these test not fail?